app/views/partials/lib/commands/posts/create/build.liquid

```diff
@@ -6,6 +6,7 @@
     "title": {{ object.title | json }},
     "body": {{ object.body | json }},
     "tags": {{ object.tags | downcase | split: ',' | uniq }},
+    "uuid": {{ '' | uuid | json }},
```

Comment:
```suggestion
    "uuid": "{{ '' | uuid }}",
```

Reply:
@Slashek Why?

Reply:
@piotrze why would you want to use `json` filter?

Reply:
@Slashek I don't have to think how to represent `string` in json. Also I don't have to think of the variable type.